packages/harpguru-core/src/components/harp-cell-accessible/hooks/use-tap-animation-value/use-tap-animation-value.ts

```diff
@@ -0,0 +1,15 @@
+import { useTimingTransition } from 'react-native-redash'
+import { Easing, interpolate, Node } from 'react-native-reanimated'
+
+export const useTapAnimationValue = (isTapped: boolean): Node<number> => {
+  const transitionValue = useTimingTransition(isTapped, {
+    duration: 100,
```

Comment:
This duration is connected to a value in packages/harpguru-core/src/components/harp-cell/hooks/use-tap-rerender-logic/use-tap-rerender-logic.ts so we should convey that relationship somehow.

Reply:
Done